packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -41,6 +43,9 @@ const DetailsComponent = ({ message = '' }) => {
                         {blockchain_hash && index === messages.length - 1 && (
                             <Clipboard text_copy={blockchain_hash} popoverAlignment='top' />
                         )}
+                        {address_hash && action_type === 'withdrawal' && index === messages.length - 1 && (
+                            <Clipboard text_copy={address_hash} popoverAlignment='top' />
+                        )}
```

Comment:
Will it show two clipboard icons next to each other?

Reply:
hi bala, it won't